src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx

```diff
@@ -17,21 +19,13 @@ import mobileHandledExceptionGroups from 'images/mobile_screenshot-full_handled-
 
 import mobileHandledExceptionLocations from 'images/mobile_screenshot-full_handled-exception-locations.png'
 
-import mobileHandledExceptionSummary from 'images/mobile_screenshot-full_handled-exception-summary.png'
+import mobileHandledExceptionsOccurrences from 'images/mobile_screenshot-full_handled-exception-occurrences.png'
 
-[**Handled exceptions**](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/introduction-mobile-handled-exceptions) help you identify significant factors contributing to poor mobile application experience, and use filterable data to find a resolution more quickly. You can also use the [handled exceptions API](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/introduction-mobile-handled-exceptions#hex-api) to customize the data you send, and use NRQL to query and share the data.
+import mobileHandledExceptionLocationExample from 'images/mobile_screenshot-full_handled-exception-location-example.png'
 
-## Handled exceptions workflow [#hex-ui]
+import mobileEventTrailHandledExceptions from 'images/mobile_screenshot-full_event-trail-handled-exceptions.png'
 
-To get the most out of the **Handled exceptions** UI, use this basic workflow:
-
-1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Exceptions > Handled exceptions**.
-2. Use any of New Relic's [standard page functions](/docs/data-analysis/user-interface-functions/view-your-data/standard-page-functions) to drill down into detailed information; for example, zoom into any area of a chart.
-3. Look for obvious or general trends in the **Users affected** and **Sessions affected** [percentage charts](#percentage-charts).
-4. Adjust the types of exceptions shown by using [groups and filters](#groups-filters).
-5. Optional: [Query or share the chart data](#insights).
-6. Look for similar patterns where exceptions appear in stack traces with the [**Top 5 exception locations** table](#top5-exceptions).
-7. To view stack trace [thread details for each occurrence](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-occurrences) of the exception, select a record from the **Top 5 exceptions location** table.
+Our **Handled exceptions** feature help you identify significant factors contributing to poor mobile application experience, and use filterable data to find a resolution more quickly. You can also use the [handled exceptions API](#hex-api) to customize the data you send, and use NRQL to query and share the data.
```

Comment:
I think "handled exceptions" should be lowercase. 
https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage/

Reply:
@paperclypse But it's the name of the UI... still think it should be lowercased?
<img width="605" alt="image" src="https://user-images.githubusercontent.com/42753584/228020988-ca3ce790-f331-4dea-8273-30847ebb1e15.png">

Reply:
Yeah, it should still be lowercased unless you are specifically referring to the UI in a procedure, such as "click *Handled exceptions*, then such and such"

When just referring to it more broadly, it should be lowercase

Reply:
Ah I see. Okay I'll fix it in another PR.